packages/analytics-js/__tests__/services/StoreManager/component-cookie/index.test.ts

```diff
@@ -51,9 +51,11 @@ describe('cookie(name, null)', () => {
     expect('0').toEqual(obj.name);
   });
 
-  it('should ignore URIError and return null', () => {
+  it('should throw URIError for malformed cookie data', () => {
```

Comment:
seems like we changed the behaviour here. this could be breaking change

Reply:
Yes. There is sufficient error handling in the logic in the above layers. So, we don't need to handle any errors here.
This should be fine, as we're not using component-cookie outside of the storage modules.

I've verified that there is no breaking change here.